etna/transforms/embeddings/models/ts2vec.py

```diff
@@ -239,7 +258,7 @@ def save(self, path: pathlib.Path):
 
     @classmethod
     def load(cls, path: pathlib.Path) -> "TS2VecEmbeddingModel":
-        """Load an object.
+        """Load an object. Model's weights are transferred to cpu during loading.
```

Comment:
This clarification about CPU should go in extended summary on next line after empty line.